### ISSUE TITLE
SA1413 incorrectly fires when items are in single lines but braces are on a separate line

### ISSUE THREAD

**Issue by alefranz** [DotNetAnalyzers/StyleCopAnalyzers#3193]
The rule
```
SA1413	Use trailing comma in multi-line initializers
```
fires incorrectly when all elements are on a single line but curly braces are not.

## Example
Example from the ASP.NET Core API template

```csharp
private static readonly string[] Summaries = new[]
{
    "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
};
```

## Expected behaviour

The rule does not fires if all element are on the same line, independently from braces/brackets position

**Comment by sharwell** [comment]
This rule is expected to fire in all cases where the last item in the list is not on the same line as the `}`.
